### PR TITLE
pre_bump_release: make sure base_release is always a string

### PR DESCRIPTION
**Pull request by vrutkovs**
/cc @owtaylor

### CODE REVIEW

File: tests/plugins/test_bump_release.py [hunk 1]
```diff
@@ -215,8 +218,9 @@ def getBuild(self, build_info):
         labels = {
             'com.redhat.component': 'component1',
             'version': 'fc26',
-            'release': base_release
         }
+        if base_release:
```
Comment (rcerven) [inline tests/plugins/test_bump_release.py]: why this if? 

if base_release will be None, it will be set to '0'

Comment (vrutkovs) [reply tests/plugins/test_bump_release.py]: Without this block the tests would pass `base_release` as a string `'None'`, but not `None`

Comment (rcerven) [reply tests/plugins/test_bump_release.py]: ahh ok

File: atomic_reactor/plugins/pre_bump_release.py [hunk 2]
```diff
@@ -90,9 +90,10 @@ def get_next_release_append(self, component, version, base_release):
         # This is brute force, but trying to use getNextRelease() would be fragile
         # magic depending on the exact details of how koji increments the release,
         # and we expect that the number of builds for any one base_release will be small.
+        release = base_release or '0'
```
Comment (lcarva) [inline atomic_reactor/plugins/pre_bump_release.py]: Should it be 1 instead? At least that's what Koji does.

Comment (vrutkovs) [reply atomic_reactor/plugins/pre_bump_release.py]: Oh, agreed. Fixed

**Comment by lcarva** [comment]
Not updating release notes draft because this is bugfix to an unreleased feature.

**Pull request merged**